deepspeed/pt/deepspeed_config.py

```diff
@@ -421,13 +421,13 @@ def _do_sanity_check(self):
         self._do_warning_check()
 
     def print(self, name):
-        print('{}:'.format(name), flush=True)
+        logger.info('{}:'.format(name))
         for arg in sorted(vars(self)):
             if arg != '_param_dict':
                 dots = '.' * (29 - len(arg))
-                print('  {} {} {}'.format(arg, dots, getattr(self, arg)), flush=True)
+                logger.info('  {} {} {}'.format(arg, dots, getattr(self, arg)))

```

Comment:
I think these logs are problematic and the logging prefix destroys formatting.

Reply:
As long as we append the same prefix, the style is still consistent?

Reply:
Agreed, the issue was that it was double-printing (now fixed I see) and since they had different prefixes, it was no longer readable. Here's the new result:

![image](https://user-images.githubusercontent.com/620322/83809515-65ed8d80-a66b-11ea-9c01-bc75ae164bbe.png)